spec/system/candidate_interface/candidate_entering_interview_preferences.rb

```diff
@@ -0,0 +1,100 @@
+require 'rails_helper'
+
+RSpec.feature 'Entering interview preferences' do
+  include CandidateHelper
+
+  scenario 'Candidate submits their interview preferences' do
+    given_i_am_not_signed_in
+    and_i_visit_the_interview_preferences_page
+    then_i_should_see_the_homepage
+
+    given_i_am_signed_in
+    and_i_visit_the_site
+
+    when_i_click_on_interview_preferences
+    and_i_submit_the_form
+    then_i_should_see_validation_errors
+
+    when_i_fill_in_an_answer
+    and_i_submit_the_form
+    then_i_can_check_my_answers
+
+    when_i_click_to_change_my_answer
+    and_i_fill_in_a_different_answer
+    and_i_submit_the_form
+    then_i_can_check_my_revised_answers
+
+    when_i_submit_my_interview_preferences
+    then_i_should_see_the_form
+    and_that_the_section_is_completed
+
+    when_i_click_on_interview_preferences
+    then_i_can_check_my_revised_answers
+  end
+
+  def given_i_am_not_signed_in; end
+
+  def given_i_am_signed_in
+    create_and_sign_in_candidate
+  end
+
+  def and_i_visit_the_interview_preferences_page
+    visit candidate_interface_interview_preferences_edit_path
+  end
+
+  def then_i_should_see_the_homepage
+    expect(page).to have_current_path(candidate_interface_start_path)
+  end
+
+  def and_i_visit_the_site
+    visit candidate_interface_application_form_path
+  end
+
+  def when_i_click_on_interview_preferences
+    click_link t('page_titles.interview_preferences')
+  end
+
+  def and_i_submit_the_form
+    click_button t('application_form.personal_statement.interview_preferences.complete_form_button')
+  end
+
+  def when_i_click_to_change_my_answer
+    first('.govuk-summary-list__actions').click_link 'Change'
+  end
+
+  def when_i_fill_in_an_answer
+    scope = 'application_form.personal_statement'
+    fill_in t('interview_preferences.label', scope: scope), with: 'Hello world'
+  end
+
+  def then_i_can_check_my_answers
+    expect(page).to have_content 'Your knowledge about the subject you want to teach'
```

Comment:
As we're making an expectation on the heading, maybe we should use the translation string?